aepsych/utils.py

```diff
@@ -277,3 +275,12 @@ def get_dim(config) -> int:
             )  # Choice parameters with n_choices < 3 add n_choices - 1 dims
 
     return dim
+
+
+def ignorable(func):
```

Comment:
Can you document what this wrapper does? And it looks like it should maybe be in some sort of `tests/utils.py` file or `tests/common.py` rather than in here?